process-engine/bpmn-studio
# ✨ Improve FlowNode Selection in Inspect Correlation

## Changes

1. Select Previously Selected FlowNode or StartEvent
2. Add FlowNode Selection Via Arrow Keys

## Issues

Closes #1071

PR: #1657

## How to test the changes

- Go to the inspect correlation
- Select an element
- Switch to another process instance with the same element
- **Notice that the element will still be selected.**

- Switch to another process instance without the same element
- **Notice that the start element will be selected.**

- Use the arrow keys
- **Note that the selected element changes depending on which arrow key is pressed.**
